Tests/Common.ps1

```diff
@@ -33,10 +33,15 @@ function Initialize-CommonTestSetup
     param()
 
     $moduleRootPath = Split-Path -Path $PSScriptRoot -Parent
-    . (Join-Path -Path $moduleRootPath -ChildPath 'Tests\Config\Settings.ps1')
+    $settingsPath = Join-Path -Path $moduleRootPath -ChildPath 'Tests\Config\Settings.ps1'
+    . $settingsPath
     Import-Module -Name (Join-Path -Path $moduleRootPath -ChildPath 'PowerShellForGitHub.psd1') -Force
 
-    if ([string]::IsNullOrEmpty($env:ciAccessToken))
+    $originalSettingsSha256Hash = "944D5BB450AD2C49F77DAE6C472FEF774B108CD2CB6A0DBF45EF4BBB7FE25D56"
+    $currentSettingsSha256Hash = (Get-FileHash -Path $settingsPath -Algorithm SHA256).Hash
+    $isSettingsUnaltered = $originalSettingsSha256Hash -eq $currentSettingsSha256Hash
```

Comment:
```suggestion
    $settingsIsUnaltered = $originalSettingsSha256Hash -eq $currentSettingsSha256Hash
```

PowerShell naming conventions follow the .NET naming conventions.  A `[bool]` function should ideally be named as a question (e.g. `IsSettingsAltered`), but a variable isn't asking a question, it's _answering_ a question, so it should be declarative in what value it holds.

Reply:
Thanks again! I went with `$settingsAreUnaltered`; I hope that was alright.